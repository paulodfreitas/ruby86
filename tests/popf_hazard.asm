.module m0
.pseg
    irmovl $100, %esp
    pushf
	subl %eax, %eax
	popf
	je if
else:
    irmovl $1, %ecx
    jmp end
if:
    irmovl $0, %ecx
end:
	halt
.end
