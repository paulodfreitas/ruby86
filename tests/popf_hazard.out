@ 0000
30
84
00
01
00
00
a0
00
61
00
b0
00
73
1c
00
00
00
30
81
01
00
00
00
70
22
00
00
00
30
81
00
00
00
00
10
@ 0023
